docs/en_US/nas.rst

```diff
@@ -1,6 +1,6 @@
-##########################
-Neural Architecture Search
-##########################
+#####################################
+Neural Architecture Search (Retiarii)
+#####################################
 
```

Comment:
Retiarii for Neural Architecture Search (NAS)

Reply:
updated